docs/cookbook/recipe_sortable_listing.rst

```diff
@@ -104,42 +105,31 @@ Now you can update your ``services.yaml`` to use the handler provided by the ``p
         app.admin.client:
             class: App\Admin\ClientAdmin
             tags:
-                - { name: sonata.admin, model_class: App\Entity\Client, controller: 'PixSortableBehaviorBundle:SortableAdmin', manager_type: orm, label: 'Clients' }
+                - { name: sonata.admin, model_class: App\Entity\Client, controller: 'SonataAdminBundle:CRUD', manager_type: orm, label: 'Clients' }
```

Comment:
```suggestion
                - { name: sonata.admin, model_class: App\Entity\Client, manager_type: orm, label: 'Clients' }
```

Reply:
understood, because `sonata.admin.configuration.default_controller=sonata.admin.controller.crud`
![image](https://user-images.githubusercontent.com/661654/189280910-3057fa42-c1f6-4525-a378-de9821b26c95.png)